github/config.go

```diff
@@ -51,6 +51,7 @@ func (c *Config) PromptForHost(host string) (h *Host, err error) {
 	h = c.Find(host)
 	if h != nil {
 		if h.User == "" {
+			utils.Check(newConfigService().CheckWriteable(configsFile()))
```

Comment:
There's very little need that `CheckWriteable` is a method on `configService`. How about we make it a standalone function so we don't have to instantiate a dummy `configService` object to access the method?